generated/nidaqmx/_task_modules/channels/ai_channel.py

```diff
@@ -4997,11 +4997,10 @@ def ai_impedance(self):
             self._handle, self._name, ctypes.byref(val))
         check_for_error(error_code)
 
-        return Impedance1(val.value)
+        return val.value
 
     @ai_impedance.setter
     def ai_impedance(self, val):
-        val = val.value
         cfunc = lib_importer.windll.DAQmxSetAIImpedance
```

Comment:
In addition to @zhindes 's last comment, the line that dereferences val.value is gone.